resources/lcd-image-converter-zh_CN.ts

```diff
@@ -0,0 +1,2287 @@
+        <translation>图像 (*.bmp *.gif *.jpg *.jpeg *.png *.pbm *.pgm *.ppm *.tiff *.xbm *.xpm)</translation>
+    </message>
+    <message>
+        <location filename="../classes/action-handlers/actionfilehandlers.cpp" line="146"/>
+        <source>Rename</source>
+        <translation>重命名</translation>
+    </message>
+    <message>
+        <location filename="../classes/action-handlers/actionfilehandlers.cpp" line="147"/>
+        <source>New name:</source>
+        <translation>新名称:</translation>
+    </message>
+    <message>
+        <location filename="../classes/action-handlers/actionfilehandlers.cpp" line="181"/>
+        <source>Save file as</source>
+        <translation>文件另存为</translation>
+    </message>
+    <message>
+        <location filename="../classes/action-handlers/actionfilehandlers.cpp" line="455"/>
+        <source>Save result file as</source>
+        <translation>结果文件另存为</translation>
+    </message>
+    <message>
+        <location filename="../classes/action-handlers/actionfilehandlers.cpp" line="458"/>
+        <source>C source code (*.c)</source>
+        <translation>C 源代码 (*.c)</translation>
+    </message>
+    <message>
+        <location filename="../classes/action-handlers/actionfilehandlers.cpp" line="459"/>
+        <source>C++ source code (*.cpp)</source>
+        <translation>C++ 源代码 (*.cpp)</translation>
+    </message>
+    <message>
+        <location filename="../classes/action-handlers/actionfilehandlers.cpp" line="460"/>
+        <source>C/C++ headers (*.h)</source>
+        <translation>C/C++ 头文件 (*.h)</translation>
+    </message>
+    <message>
+        <location filename="../classes/action-handlers/actionfilehandlers.cpp" line="461"/>
+        <source>C/C++ files (*.c *.cpp *.h)</source>
+        <translation>C/C++ 文件 (*.c *.cpp *.h)</translation>
+    </message>
+    <message>
+        <location filename="../classes/action-handlers/actionfilehandlers.cpp" line="462"/>
+        <source>All Files (*.*)</source>
+        <translation>所有文件 (*.*)</translation>
+    </message>
+</context>
+<context>
+    <name>AppUI::Setup::DialogExternalEditor</name>
+    <message>
+        <location filename="../controls/setup/dialogexternaleditor.cpp" line="60"/>
+        <source>All files (*.*)</source>
+        <translation>所有文件 (*.*)</translation>
+    </message>
+    <message>
+        <location filename="../controls/setup/dialogexternaleditor.cpp" line="61"/>
+        <source>Select application</source>
+        <translation>选择应用程序</translation>
+    </message>
+</context>
+<context>
+    <name>AppUI::Setup::DialogOptions</name>
+    <message>
+        <location filename="../controls/setup/dialogoptions.cpp" line="218"/>
+        <source>Import - Attention</source>
+        <translation>导入 - 注意</translation>
+    </message>
+    <message>
+        <location filename="../controls/setup/dialogoptions.cpp" line="262"/>
+        <source>Enter preset name</source>
+        <translation>输入预设名称</translation>
```

Comment:
Translator shows this as "Enter the default name".
But 'preset' here means template, collection of predefined settings, one of many, not default.
Is it ok?